compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -290,6 +299,16 @@ class ROCMTargetBackend final : public TargetBackend {
     pb.registerFunctionAnalyses(fam);
     pb.registerLoopAnalyses(lam);
     pb.crossRegisterProxies(lam, fam, cgam, mam);
+    // Attempt to load pass plugins and register their callbacks with PB.
+    for (auto &PluginFN : passPlugins) {
```

Comment:
mlir uses thisCaseForLocalVariableNames. Also don't use auto then the type is not obvious based on the RHS only.

```suggestion
    for (StringRef pluginFn : passPlugins) {
```